common/source/docs/common-archived-topics.rst

```diff
@@ -49,6 +49,9 @@ value to users with old hardware.
     Copter Project News <project-news>
     Troubleshooting <troubleshooting>
     Landing Gear <landing-gear>
+    [/site]
+[site wiki="rover"]
+    Old Rover Avoidance <rover-object-avoidance>
```

Comment:
Let's make "Old Rover avoidance" specify the version.. so "Rover Avoidance 3.5.x and earlier"

Reply:
done